lib/src/constants/type_defs.dart

```diff
@@ -58,3 +58,5 @@ typedef PreviewTransformBuilder = Widget? Function(
   CameraController controller,
   Widget child,
 );
+
+typedef StopVideoRecordingCallback = void Function(XFile);
```

Comment:
Use the above comments as references, to write a more specific comment.

Reply:
Done